airbyte-integrations/connectors/source-personio/source_personio/schemas/employees.json

```diff
@@ -0,0 +1,85 @@
+{
```

Comment:
Can the schema for employees differ?
What happens if the schema is different?

Reply:
@alexander-pospiech-parkdepot Yes, it can! This step is a bit tricky because of data privacy problems:
- If a column is added, we simply won't see
- If a column is deleted, we will start to have only `null` values from that point on, but we will not get any errors.

In theory, we can improve this step by having an automatic schema definition. But I'm a bit resistant to doing this now, mainly because on Airbyte we cannot decide which columns to load, only which tables. The problem is that this endpoint `company/employees` contain columns such as `salary` -> Which is a piece of problematic information to have in our Warehouse without a good privacy strategy implemented.

The alternative that I thought, of is to implement a `discovery_schema` function to get this automatic schema, and add a transformation step on Airbyte to remove the problematic columns. The downside: we would have this information temporarily anyway on the airbyte temp tables (which they destroy after finishing the load successfully). I really don't know if this is bad or not, so decided for now to have this static schema.

This step is the only reason why I think this MR is not ready to be added to the original airbyte repository, otherwise this could be a perfect contribution.